docs/sql-guide/statements/statement-select.md

```diff
@@ -0,0 +1,186 @@
+---
+title: SELECT
+layout: default
+parent: Statements
+grand_parent: SQL guide
+---
+
+# SELECT statement
+
+Selects data from a FeatureBase table.
+
+## BNF diagrams
+
+![expr](/assets/images/sql-guide/select_stmt.svg)
+
+### top_clause
+![expr](/assets/images/sql-guide/top_clause.svg)
+
+### select_list
+![expr](/assets/images/sql-guide/select_list.svg)
+![expr](/assets/images/sql-guide/select_item.svg)
+
+### from_clause
+![expr](/assets/images/sql-guide/from_clause.svg)
+![expr](/assets/images/sql-guide/table_or_subquery.svg)
+![expr](/assets/images/sql-guide/table_option.svg)
+
+### where_clause
+![expr](/assets/images/sql-guide/where_clause.svg)
+
+### group_by_clause
+![expr](/assets/images/sql-guide/group_by_clause.svg)
+
+### having_clause
+![expr](/assets/images/sql-guide/having_clause.svg)
+
+### order_by_clause
+![expr](/assets/images/sql-guide/order_by_clause.svg)
+![expr](/assets/images/sql-guide/order_by_expression.svg)
+
+## DDL Syntax
+
+```
+SELECT
```

Comment:
Looks a little strange compared to other syntaxes like ALTER, CREATE, etc. Can catch later but just noting